integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template

```diff
@@ -71,7 +71,9 @@ spec:
               cpu: 1
               memory: 1G
           command: ["/entrypoint.sh"]
-          args: ["master-only", "--no-format"]
+          args:
```

Comment:
Do we explicitly prefer this expression over the `["a", "b"]` way?

Reply:
nope, i just made it consistent w/ what we had in the fuse template previously. its slightly easier to add new args in `config.yaml` using the newline way

Reply:
SGTM